writing-adapters.md

```diff
@@ -0,0 +1,164 @@
+# Writing database adapters
+
+`micro-analytics` database adapters are simple JavaScript modules which export an object with five methods. They _have_ to be called `micro-analytics-adapter-xyz`, where `xyz` is the name users will pass to the `DB_ADAPTER` environment variable when starting `micro-analytics`.
```

Comment:
I wouldn't bother specifying the number of methods anywhere. That can get out of sync quick. Probably just mention that it needs to implement the minimal interface described below.